static/js/drafts.js

```diff
@@ -410,6 +411,7 @@ export function launch() {
         if ($("#drafts_table .draft-row").length > 0) {
             $("#drafts_table .no-drafts").hide();
         }
+        rendered_markdown.update_elements($("#drafts_table"));
```

Comment:
We should call this on the portion that has the `rendered_markdown` CSS class -- anything outside that is.  E.g. from the info overlay:
```
    $markdown_help.find(".rendered_markdown").each(function () {                                                                
        rendered_markdown.update_elements($(this));                                                                             
    });                                                                                                                         
```

See also https://github.com/zulip/zulip/pull/18234/commits/4c698ca38105ac659805257bac7e026e8a998c00 which was a previous PR for this by @m-e-l-u-h-a-n, that he hasn't had a chance to finish.  Perhaps you can pick it up?

Reply:
Got it. I replaced that line (locally) with:
```
        $("#drafts_table")
            .find(".rendered-markdown")
            .each(function () {
                rendered_markdown.update_elements(this);
            });
```
However, this is again making the node test `draft.js` fail at format_drafts() due to the usage of `find()`. Any suggestions on how to fix the tests? Here's the logs:
```
Starting node tests...
running test drafts
        test: legacy
        test: draft_model add
        test: draft_model edit
        test: draft_model delete
        test: snapshot_message
        test: initialize
        test: remove_old_drafts
        test: update_draft
        test: delete_all_drafts
        test: format_drafts
--------------------------------------------------
test failed: /home/nehal/zulip/frontend_tests/node_tests/drafts.js > format_drafts

Error: 
                We need you to simulate the results of $(...).find(...)
                by using set_find_results. You want something like this:

                    const container = ...;
                    const child = ...;
                    container.set_find_results(".rendered-markdown", child);

                Then calling container.find(".rendered-markdown") will return
                the "child" zjquery element.

                
    at Proxy.find (/home/nehal/zulip/frontend_tests/zjsunit/zjquery_element.js:83:19)
    at render_widgets (/home/nehal/zulip/static/js/drafts.js:415:14)
    at Object.launch (/home/nehal/zulip/static/js/drafts.js:442:5)
    at /home/nehal/zulip/frontend_tests/node_tests/drafts.js:446:12
    at /home/nehal/zulip/frontend_tests/node_tests/drafts.js:111:9
    at /home/nehal/zulip/frontend_tests/zjsunit/test.js:34:13
    at Object.exports.with_overrides (/home/nehal/zulip/frontend_tests/zjsunit/namespace.js:490:9)
    at exports.run_test (/home/nehal/zulip/frontend_tests/zjsunit/test.js:33:19)
    at test (/home/nehal/zulip/frontend_tests/node_tests/drafts.js:108:5)
    at Object.<anonymous> (/home/nehal/zulip/frontend_tests/node_tests/drafts.js:324:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module._compile (/srv/zulip-npm-cache/2067d95fb0f383a045d739b3b31c546943524781/node_modules/pirates/lib/index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.newLoader [as .js] (/srv/zulip-npm-cache/2067d95fb0f383a045d739b3b31c546943524781/node_modules/pirates/lib/index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.load [as _load] (/home/nehal/zulip/frontend_tests/zjsunit/namespace.js:90:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at run_one_module (/home/nehal/zulip/frontend_tests/zjsunit/index.js:78:5)
(...)


** Tests failed, PLEASE FIX! **
```